static/js/nodes.js

```diff
@@ -9,27 +9,38 @@
     url += '?vhost=' + encodeURIComponent(vhost)
   }
   let updateTimer = null
-  let data = null
-
-  if (data === null) {
-    update(render)
-  }
 
   function update (cb) {
-    lavinmq.http.request('GET', url).then(function (response) {
-      data = response
-      render(response)
+    const statsPromise = lavinmq.http.request('GET', url)
+    const queueStatsPromise = lavinmq.http.request('GET', '/api/queues')
```

Comment:
We might want to do this sum server side, if we have a lot of queues. Maybe we can do an experiment with 10k queues how long this takes here? :)

Reply:
Yes true! Ill try putting it server side and re-iterate 👍